qjs.c

```diff
@@ -482,6 +545,39 @@ int main(int argc, char **argv)
                 stack_size = parse_limit(opt_arg);
                 break;
             }
+            if (opt == 'c' || !strcmp(longopt, "compile")) {
+                if (!opt_arg) {
+                    if (optind >= argc) {
+                        fprintf(stderr, "qjs: missing file for -c\n");
+                        exit(2);
```

Comment:
For my curiosity: why 2 as the exit code?

Reply:
C&P, I'm changing it to 1 like the other exit codes. There are 2 left that exit with code 2, when we fail to allocate the runtime or context. Do we care about them being different?